### PR TITLE
LPS-176172 Set HTTPS scheme only if `Portal.isSecure` method returns true, otherwise the value keeps unaltered from the getBaseUriBuilder method

### PR DESCRIPTION
**Pull request by ccorreagg**
This PR contains the code to fix the regression bug https://issues.liferay.com/browse/LPS-176172.

The value of the UriBuilder from the invocation `UriInfoUtil.getBaseUriBuilder(uriInfo)` already contains the scheme set to `HTTPS` if one of the properties `portal.instance.protocol` or `web.server.protocol` are set to `true`.

### CODE REVIEW

**Comment by ccorreagg** [comment]
ci:test:sf

**Comment by ccorreagg** [comment]
ci:test:relevant

File: modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/resource/OpenAPIResourceImpl.java [hunk 1]
```diff
@@ -518,20 +518,14 @@ private String _getBasePath(
...
 		}
 
-		String scheme = Http.HTTP;
+		UriBuilder uriBuilder = UriInfoUtil.getBaseUriBuilder(uriInfo);
+
+		uriBuilder.host(_portal.getForwardedHost(httpServletRequest));
 
 		if (_portal.isSecure(httpServletRequest)) {
-			scheme = Http.HTTPS;
+			uriBuilder.scheme(Http.HTTPS);
```
Comment (LuismiBarcos) [inline modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/resource/OpenAPIResourceImpl.java:526]: I assume that scheme is `HTTP` by default right?

**Review by LuismiBarcos — APPROVED** [review]
LGTM :) 👏 👏 👏 👏 
Just a doubt but seems to be OK

File: modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/resource/OpenAPIResourceImpl.java (hunk 1 above)
Comment (ccorreagg) [reply modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/resource/OpenAPIResourceImpl.java:526]: I assume we should not change, so it is managed by the UriInfoUtil in this method https://github.com/liferay/liferay-portal/blob/7a541f63ee4b774288dbc2673142961c493a059d/modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/util/UriInfoUtil.java#L44

**Comment by ccorreagg** [comment]
ci:forward

**Pull request closed without merging**